app/models/merchant.rb

```diff
@@ -4,4 +4,16 @@ class Merchant < ApplicationRecord
   has_many :customers, through: :invoices
   has_many :invoice_items, through: :invoices
   has_many :transactions, through: :invoices
+
+  def self.all_items(id)
```

Comment:
I don't think you should need to pass ids to class methods. Self should be good enough...pretty sure